python/sglang/srt/managers/schedule_batch.py

```diff
@@ -1620,6 +1633,14 @@ class ModelWorkerBatch:
     # If set, the output of the batch contains the hidden states of the run.
     capture_hidden_mode: CaptureHiddenMode = None
 
+    # HPU-specific fields
```

Comment:
possible to call this is paged_allocator specific fields?

Reply:
I think we should pack these fields as a dataclass in hpu_utils.py